W-15925139-Snapshot-asset-support (#2730)
mulesoft/docs-mule-runtime: modules/ROOT/pages/_partials/mmp-deploy-to-cloudhub-2.adoc

mvn clean deploy -DmuleDeploy
----

> === Maven Snapshot Assets

Reviewer: Let us change terminology as Exchange Snapshot*

> === Maeven Snapshot Asssets

Reviewer 2: ```suggestion
=== Maven Snapshot Assets
```

> You can also deploy Maven snapshot assets into CloudHub 2.0. To learn more about publishing snapshot assets to Anypoint Exchange, see xref:exchange::to-publish-assets-maven.adoc#asset-lifecycle-state[Asset Lifecycle State].

Reviewer: Same as above, let us call it Exchange snapshots (since even exchange docs does not mention 'maven snapshots'

[NOTE]

> Each time you update your application's snapshot, you need to redeploy the application to refresh it with the latest snapshot binaries. 

Reviewer 2: ```suggestion
Each time you update your application's snapshot, redeploy the application to refresh it with the latest snapshot binaries. 
```

+

> Since snapshot assets can change after they are deployed, avoid deploying them into your production environment.

Reviewer 2: ```suggestion
Since snapshot assets can change after deployment, avoid deploying them into your production environment.
```

mvn clean deploy -DmuleDeploy
----

=== Exchange Snapshot Assets

You can also deploy Exchange snapshot assets into CloudHub 2.0. 

> By using `SNAPSHOT` version assets in Anypoint Exchange during the development and testing phase, you can avoid incrementing your application's version number for small changes. Once your `SNAPSHOT` version application has been overwritten in Anypoint Exchange, you can redeploy your `SNAPSHOT` version application to CloudHub 2.0 via the Mule Maven plugin to deploy the latest changes.

Reviewer 3: ```suggestion
By using `SNAPSHOT` version assets in Anypoint Exchange during the development and testing phase, you can avoid incrementing your application's version number for small changes. After your `SNAPSHOT` version application has been overwritten in Anypoint Exchange, you can redeploy your `SNAPSHOT` version application to CloudHub 2.0 via the Mule Maven plugin to deploy the latest changes.
```

To learn more about publishing snapshot assets to Anypoint Exchange, see xref:exchange::to-publish-assets-maven.adoc#asset-lifecycle-state[Asset Lifecycle State].

[NOTE]
Each time you update your application's snapshot, redeploy the application to refresh it with the latest snapshot binaries. 
+

> Since snapshot assets can change after deployment, avoid deploying them into your production environment.

Reviewer 3: ```suggestion
Because snapshot assets can change after deployment, avoid deploying them into your production environment.
```
